internal/upgrader/upgrader.go

```diff
@@ -32,12 +34,10 @@ type Logger interface {
 }
 
 type UpgradeConfig struct {
-	BrokerName            string
-	ParallelUpgrades      int
-	DryRun                bool
-	MinVersionRequired    string
-	CheckUpToDate         bool
-	CheckDeactivatedPlans bool
+	BrokerName       string
+	ParallelUpgrades int
+	Action           config.Action
```

Comment:
question: we are reversing the order of dependencies, now upgrade depends on config. Perhaps we should establish our actions in the upgrader. What do you think?